content/docs/components/multi-tenancy/getting-started.md

```diff
@@ -104,31 +53,31 @@ presented with an error.
   alt="Error listing notebooks in inacessible profile"
   class="mt-3 mb-3 border border-info rounded">
 
-When the users create Jupyter notebook servers from the Notebooks Servers UI,
-the notebook pods are created in the active profile. If the user doesn't have
-modify access to the active profile, the user can only browse currently active
+When you create Jupyter notebook servers from the Notebooks Servers UI,
+the notebook pods are created in your active profile. If you don't have
+modify access to the active profile, you can only browse currently active
 notebook servers and access the existing jupyter notebooks but cannot create
-new notebook servers in that profile. The users can create notebook
-servers in their primary profile which they have view and modify access to.
+new notebook servers in that profile. You can create notebook
+servers in your primary profile which they have view and modify access to.
 
 ## Onboard new user
 
-Kubeflow v0.6.2 provides automatic profile creation for authenticated users on
+Kubeflow {{% kf-latest-version %}} provides automatic profile creation for authenticated users on
 first login. Additionally, an **administrator** can create a profile for any
 user in the kubeflow cluster.  Here an administrator is a person who has
 [*cluster-admin*](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#user-facing-roles)
 role binding in the Kubernetes cluster. This person has permissions to create
 and modify Kubernetes resources in the cluster. For example, the person who
 deployed Kubeflow will have administration privileges in the cluster.
 
-### Pre-requisite: Grant user minimal Kubernetes cluster view access
+### Pre-requisites: grant user minimal Kubernetes cluster view access
 
 User should be given minimal permission scope that allows them to connect to the Kubernetes cluster.
 For example for GCP users, they can be granted IAM roles: **Kubernetes Engine Cluster Viewer** and **IAP-secured Web App User**
 
-### Automatic creation of Profiles
+### Automatic creation of profiles
 
-Kubeflow v0.6.2 onwards provides automatic profile creations as a convenience
+Kubeflow {{% kf-latest-version %}} onwards provides automatic profile creations as a convenience
```

Comment:
I think we should remove the word "onwards" now that we're using a variable for the version. The word "onwards" implies that this is the first version that provided this feature.